frame/staking/src/tests.rs

```diff
@@ -2778,12 +2778,91 @@ fn deferred_slashes_are_deferred() {
 		assert_eq!(Balances::free_balance(11), 1000);
 		assert_eq!(Balances::free_balance(101), 2000);
 
+		let _ = staking_events_since_last_call();
```

Comment:
why this is better than `System::reset_events()`?

Reply:
same thing actually, I didn't know about `reset_events`.